WordPress/Classes/Models/AbstractPost.m

```diff
@@ -292,12 +294,20 @@ - (AbstractPost *)latest
 
 - (AbstractPost *)revision
 {
-    return [self primitiveValueForKey:@"revision"];
+    [self willAccessValueForKey:@"revision"];
```

Comment:
I did try replacing the read-only getter with a read/write and drop the getter code. For some reason it majorly screwed up the filtering in the post list view controller. I didn't have time to dig down that rabbit hole so I reverted that change and put the KVC method calls in here.